aodncore/pipeline/handlerbase.py

```diff
@@ -206,6 +206,8 @@ def __init__(self, input_file,
         self._handler_run = False
         self._instance_working_directory = None
         self._notify_list = None
+        self._should_notify = None
```

Comment:
I think the purpose of these two (`_notify_list` and `_should_notify`) could be made a bit clearer by calling them (`_have_notifed_list` and `_should_notify_list`)

Reply:
Agreed, this is too ambiguous. I've left "should_upload", as the list of people who *should be* (or should have been) notified, and renamed notify_list to "notification_results", since that is actually what it is... a dict to store the results of the notification attempts.